.travis/runcollection.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+# SPDX-License-Identifier: MIT
+
+set -e
+
+#uncomment if you use $ME - otherwise set in utils.sh
+#ME=$(basename "$0")
+SCRIPTDIR=$(readlink -f "$(dirname "$0")")
+
+. "${SCRIPTDIR}/utils.sh"
+. "${SCRIPTDIR}/config.sh"
+
+# Collection commands that are run when `tox -e collection`:
+role=$(basename "${TOPDIR}")
+toxworkdir=${1:-"${TOPDIR}"/.tox}
+STABLE_TAG=${2:-master}
+cd "${toxworkdir}"
+toxworkdir=$(pwd)
+envlist=${3:- "black,flake8,yamllint,py38,shellcheck"}
+automaintenancerepo=https://raw.githubusercontent.com/linux-system-roles/auto-maintenance/
+
+curl -L -o lsr_role2collection.py "${automaintenancerepo}${STABLE_TAG}"/lsr_role2collection.py
+
+python lsr_role2collection.py --src-path "${TOPDIR}/.." --dest-path "${toxworkdir}" --role "${role}" > "${toxworkdir}"/collection.out 2>&1
```

Comment:
Shouldn't some heuristic be used to determine which python interpreter to execute? "python" might not be available if only python3 is found, or it might not point to the interpreter we want to use. (I had boxes with python2 and python3, where `python` linked to python2.)

Reply:
This is being run in a tox venv, so `python` will be whatever `python` the tox venv was configured to use.

Reply:
> This is being run in a tox venv, so `python` will be whatever `python` the tox venv was configured to use.

Right. The collection test is configured to run with python 3.6 in `tox.ini`.
` 3.6: py36,coveralls,black,yamllint,shellcheck,custom,collection`
To run it manually, we recommend to use `tox` as `tox -e collection`.

Reply:
Thank you for the explanation. That is fine then.